### ISSUE TITLE
Uncaught TypeError: t.lowLevel is not a function

### ISSUE THREAD

**Issue by lazutkin** [mulesoft/api-workbench#489]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.16.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.4
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package 0.8.45

### Stack Trace

Uncaught TypeError: t.lowLevel is not a function

```
At /Users/dimka/.atom/packages/api-workbench/main.js:1

TypeError: t.lowLevel is not a function
    at t.setSelectedNode (/packages/api-workbench/main.js:1:13115)
    at /packages/api-workbench/main.js:1:21795
    at /packages/api-workbench/node_modules/atom-ui-lib/dist/UI.js:414:68
    at Array.forEach (native)
    at HTMLButtonElement.element.onclick (/packages/api-workbench/node_modules/atom-ui-lib/dist/UI.js:414:31)
```

### Commands

```
     -0:27.6.0 autocomplete-plus:confirm (input.hidden-input)
     -0:27.1.0 markdown-writer:indent-list-line (input.hidden-input)
     -0:27.1.0 snippets:next-tab-stop (input.hidden-input)
     -0:27.1.0 snippets:expand (input.hidden-input)
     -0:27.1.0 editor:indent (input.hidden-input)
     -0:26.5.0 core:backspace (input.hidden-input)
  2x -0:25 core:move-down (input.hidden-input)
     -0:24.4.0 markdown-writer:insert-new-line (input.hidden-input)
     -0:24.4.0 autocomplete-plus:confirm (input.hidden-input)
     -0:22.5.0 markdown-writer:insert-new-line (input.hidden-input)
     -0:22.5.0 autocomplete-plus:confirm (input.hidden-input)
     -0:22.2.0 core:save (input.hidden-input)
  3x -0:21.7.0 core:move-down (input.hidden-input)
 11x -0:20 core:move-up (input.hidden-input)
 10x -0:14.1.0 core:move-down (input.hidden-input)
     -0:11 api-workbench:quick-outline (input.hidden-input)
```

### Non-Core Packages

```
api-workbench 0.8.45 
atom-beautify 0.29.23 
atom-formatter-jsbeautify 1.1.2 
atom-handlebars 1.3.0 
atom-typescript 11.0.3 
atom-yamljson 0.2.4 
autocomplete-java 1.2.5 
busy-signal 1.4.1 
coffee-compile 0.25.0 
copy-filename 1.0.1 
file-icons 2.1.4 
formatter 2.12.4 
git-plus 7.7.0 
go-debug 1.3.1 
go-plus 5.5.2 
go-signature-statusbar 1.2.0 
gotests 0.3.0 
hyperclick 0.0.40 
intentions 1.1.2 
language-bazel 0.3.2 
language-cypher 0.3.0 
language-docker 1.1.7 
language-gnuplot-atom 0.2.0 
language-nunjucks 4.1.0 
language-pegjs 0.5.0 
language-plantuml 0.2.0 
last-cursor-position 0.9.2 
linter 2.1.4 
linter-docker 0.1.4 
linter-javac 1.9.4 
linter-js-yaml 1.2.7 
linter-raml 1.0.1 
linter-swagger 0.4.1 
linter-ui-default 1.2.4 
markdown-writer 2.6.4 
merge-conflicts 1.4.4 
minimap 4.26.8 
minimap-git-diff 4.3.1 
nord-atom-syntax 0.6.0 
open-vim 0.2.3 
plantuml-viewer 0.7.1 
pretty-json 1.6.1 
revert-buffer 0.5.0 
sort-lines 0.14.0 
status-bar-cursor-position 0.8.0 
sync-settings 0.8.1 
tabs-to-spaces 1.0.3 
tabularize 0.2.5 
tree-view-copy-relative-path 1.1.0 
tree-view-git-branch 0.1.2 
tree-view-git-status 1.3.1 
```

**Comment by ddenisenko** [comment]
Could you please provide RAML code to reproduce and what you were doing (looks like you clicked some button)?

**Comment by paul42** [comment]
got same error, was following guide for 'general functionality' and then did ctrl+O, opened up outline mode, clicked on an item and hit 'okay' to see what it did, caused this error.

**Comment by dreamflyer** [comment]
Duplicate of #474. Fix will be published in v0.8.46.
